zmon_worker_monitor/zmon_worker/notifications/opsgenie.py

```diff
@@ -34,6 +32,7 @@ def notify(cls,
                priority=None,
                message='',
                description='',
+               customfields=None,
```

Comment:
can we name it `custom_fields`? :)

Reply:
done, i was just trying to follow opsgenie naming convention but it's better that way :)